tests/test_sweet.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from astropy import units as u
+from astropy.utils.data import download_file
+from lightkurve.search import open as lc_read  # Will be read in 2.x
+from numpy.testing import assert_allclose
+
+from exovetter.tce import TCE
+from exovetter.vetters import Sweet
+
+
+@pytest.mark.remote_data
+def test_kepler_1026032():
+    filename = download_file('http://archive.stsci.edu/missions/kepler/lightcurves/0010/001026032/kplr001026032-2009166043257_llc.fits', cache=True)  # noqa
+
+    period = 8.46043892 * u.day
```

Comment:
Stats from https://exoplanetarchive.ipac.caltech.edu/cgi-bin/TblView/nph-tblView?app=ExoTbls&config=cumulative that @fergalm pointed me to.